exercises/practice/reverse-string/.approaches/native-pipes/snippet.txt

```diff
@@ -0,0 +1,5 @@
+reverse <- function(text) {
+  strsplit(text, "") |>
+    lapply(rev) |>
+    sapply(paste, collapse = "")
```

Comment:
```suggestion
    rev() |>
    paste(collapse = "")
```
(I think this should be the non-vectorised version since the vectorised version is a separate approach?)